assets/schema_input.json

```diff
@@ -51,15 +47,13 @@
                 "type": "string",
                 "format": "file-path",
                 "pattern": "^\\S+\\.kraken2\\.kraken2\\.classifiedreads\\.txt$",
-                "errorMessage": "Kraken2 classification results must be provided.",
-                "dependentRequired": ["kraken2_report"]
+                "errorMessage": "Kraken2 classification results must be provided."
             },
             "kraken2_taxpasta": {
                 "type": "string",
                 "format": "file-path",
                 "pattern": "^\\S+\\.tsv$",
-                "errorMessage": "Standardized kraken2 profiling with taxonomic information obtained from taxpasta.",
-                "dependentRequired": ["kraken2_report", "kraken2_result"]
+                "errorMessage": "Standardized kraken2 profiling with taxonomic information obtained from taxpasta."
```

Comment:
Same about the `dependentRequired`

Reply:
`dependentRequired` is still included in the schema, just moved to the bottom with the new format.